master/static/migrations/20231206140233_add-parent-id-task-config.tx.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tasks
+ADD parent_id text REFERENCES tasks(task_id) ON DELETE CASCADE DEFAULT(NULL);
```

Comment:
I assume we want child tasks to be deleted when parent tasks are. Doing this through table definition seem efficient. Thought?

Reply:
yeah, I think this makes sense.